x-pack/plugins/apm/public/components/app/service_groups/service_group_save/select_services.tsx

```diff
@@ -145,6 +145,7 @@ export function SelectServices({
                   'xpack.apm.serviceGroups.selectServicesForm.kql',
                   { defaultMessage: 'E.g. labels.team: "web"' }
                 )}
+                urlPersist={false}
```

Comment:
nit: WDYT about `persistKueryInUrl` or `persistInUrl`?

```suggestion
                persistKueryInUrl={false}
```